### PR TITLE
Update README for routerlicious

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
Removed missing architecture image, fixed up text that were being interpreted as HTML tags, added code fences, and fixed up some grammar.

### CODE REVIEW

File: server/routerlicious/README.md [hunk 1]
```diff
@@ -119,12 +119,6 @@ If you want to build API documentation locally, see [Building Documentation](htt
 
 ## Architecture
 
-Below is the original Routerlicious architecture diagram. The current system has slight changes from the diagram but
-largely remains unchanged. Detailed descriptions of the components are contained below as well as callouts
-to areas that have changed from the original picture. We will update the README with a more current diagram soon.
-
-![Routerlicious architecture diagram](../../docs/architecture/server/SystemArchitecture.jpg)
```
Comment (DanielRosenwasser) [inline server/routerlicious/README.md:126]: I wasn't able to find this image within the repo - let me know if it's still around.

File: server/routerlicious/README.md [hunk 2]
```diff
@@ -179,18 +173,6 @@ More details on content-addressable file systems and Git can be found at
 * https://git-scm.com/book/en/v2/Git-Internals-Plumbing-and-Porcelain
 * http://stefan.saasen.me/articles/git-clone-in-haskell-from-the-bottom-up/
 
-### Picture Errata
```
Comment (DanielRosenwasser) [inline server/routerlicious/README.md:182]: This isn't really necessary anymore if there's no image.

File: server/routerlicious/README.md [hunk 3]
```diff
@@ -269,10 +253,10 @@ By default, the service does not run locally. To run locally, first add the foll
...
+This will enable the metric writer to write to telegraf client. Then run `docker-compose -f docker-compose.yml -f docker-compose.dev.yml -f docker-compose.metric.yml up` to bring up telegraf, influxdb, and grafana containers. Navigate to "http://localhost:7000" to get grafana up and running.
 
 ## Authentication model
-Routerlicious uses a token based authentication model. Tenants are registered to routerlicious first and a secret key is generated for each tenant. Apps are expected to pass <secret-key>, <tenant-id>, and <user-info> as a signed token to routerlicious. Tenants are given a symmetric-key beforehand to sign the token.
+Routerlicious uses a token based authentication model. Tenants are registered to routerlicious first and a secret key is generated for each tenant. Apps are expected to pass `<secret-key>`, `<tenant-id>`, and `<user-info>` as a signed token to routerlicious. Tenants are given a symmetric-key beforehand to sign the token.
```
Comment (DanielRosenwasser) [inline server/routerlicious/README.md:259]: These were just showing up as blanks in the rendered markdown, so I could either have written

```md
\<secret-key>
```

or turned them into code as here.

File: server/routerlicious/README.md (hunk 1 above)
Comment (curtisman) [reply server/routerlicious/README.md:126]: Yeah. That file was moved in PR #1391 and that deleted in #3087.
The diagram is quite old and doesn't reflect the current picture. We should create a new one an d I have create #5002 to track that.  Removing it for now is good.

**Review by curtisman — APPROVED** [review]

**Pull request merged**